src/main/User/UserController.java

```diff
@@ -351,10 +353,13 @@ public static JSONObject mergeJSON(
                 Optional.empty(),
                 Optional.empty());
         Message mes = serv.executeAndGetResponse();
+        responseJSON = mes.toJSON();
         if (mes == FileMessage.SUCCESS) {
+          InputStream result = serv.getInputStream();
+          responseJSON.put("inputStream", result);
```

Comment:
If you're returning a JSON with the image inside, are you sure you can do content type image?